source/features/rgh-sponsor-button.tsx

```diff
@@ -82,19 +82,20 @@ async function suchLove({delegateTarget}: delegate.Event): Promise<void> {
 	love.remove(); // 💔
 }
 
-async function handleNewIssue(): Promise<false> {
+async function handleNewIssue(signal: AbortSignal): Promise<false> {
 	if (getRepo()!.owner !== getUsername() && !await cache.get('did-it-wiggle')) {
 		select('.btn-primary[href$="/issues/new/choose"], .btn-primary[href$="/issues/new"]')
 			?.addEventListener('mouseenter', wiggleWiggleWiggle, {
 				once: true,
+				signal,
```

Comment:
Oops, incompatible with Safari.

I think it won't be a huge issue, it's just that Safari users won't get this enhancement in this case. We just need to determine whether this causes duplicate calls a case-by-case basis.

However I think this shouldn't be covered anyway for the same reason highlighted in my first review: single elements are removed on unload regardless (unless they're outside the ajaxed container)

So let's drop this single line as well and everything like it (if I missed any)